typing_tests/pine-params.ts

```diff
@@ -607,12 +543,17 @@ await (async () => {
 				},
 			},
 			...(maybeService != null && {
-				is_a_build_of__service: {
-					$any: {
-						$alias: 'iabos',
-						$expr: {
-							iabos: {
-								service_name: maybeService,
+				$expand: {
```

Comment:
I think the intention of the test (which I realize I failed to do) was to have it as a conditional filter like:
```
			$filter: {
				release_image: {
					// ...
				},
				...(maybeService != null && {
					is_a_build_of__service: {
						$any: {
							$alias: 'iabos',
							$expr: {
								iabos: {
									service_name: maybeService,
								},
							},
						},
					},
				}),
			},
```
Great find 👍

Reply:
@otaviojacobi kind reminder to update this one as well